packages/amplify-util-mock/src/api/api.ts

```diff
@@ -88,6 +88,7 @@ export class APITest {
       await this.generateCode(context, appSyncConfig);
 
       context.print.info(`AppSync Mock endpoint is running at ${this.appSyncSimulator.url}`);
+      context.print.info(`GraphiQL IDE is available for local testing at ${this.appSyncSimulator.localhostUrl}`);
```

Comment:
```context.print.info(`AppSync Mock endpoint is running at ${this.appSyncSimulator.url}`);``` will print:
```console 
AppSync Mock endpoint is running at http://192.168.0.100:20002
```
However, ```context.print.info(`GraphiQL IDE is available for local testing at ${this.appSyncSimulator.localhostUrl}`);``` will print:
```console 
GraphiQL IDE is available for local testing at http://localhost:20002
```

The reason behind this change is that it is not possible to access the necessary Web APIs (in this case [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) for signing tokens)  on an insecure environment because of browser restrictions ([Secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts)). Therefore, we were going to either work on a HTTPS environment or use http://localhost which browsers support as trustworthy origin. I added the relevant line to print the localhost URL of GraphiQL Explorer for users to access.